src/test/kotlin/PublishLottoTest.kt

```diff
@@ -1,26 +1,30 @@
 import model.Lotto
+import model.LottoNumber
 import model.NumberGenerator
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
+import util.LottoConstants
 
 class PublishLottoTest {
-    private lateinit var lottoNumbers: List<Int>
+    private lateinit var lottoNumbers: List<LottoNumber>
     private lateinit var lotto: Lotto
     private val numberGenerator = NumberGenerator()
 
     @BeforeEach
     fun setUp() {
         val randomNumbers = numberGenerator.makeRandomNumbers()
-        lottoNumbers = numberGenerator.drawSixNumbers(randomNumbers)
+        lottoNumbers = numberGenerator.drawSixNumbers(randomNumbers).map { LottoNumber(it) }
         lotto = Lotto(lottoNumbers)
     }
 
     @Test
-    fun `로또 번호가 1에서 45사이의 숫자인지 확인`() {
-        assertThat(lottoNumbers.all { number ->
-            number in 1..45
-        }).isTrue
+    fun `로또 번호가 시작 범위에서 마지막 범위사이의 숫자인지 확인`() {
+        assertThat(
+            lottoNumbers.all {
+                it.number in LottoConstants.START_RANGE..LottoConstants.END_RANGE
+            },
+        ).isTrue
```

Comment:
`ObjectEnumerableAssert#allMatch(Predicate)`을 찾아보고 그 기능을 사용해 보면 어떨까요?

```suggestion
        assertThat(lottoNumbers).allMatch { it.number in LottoConstants.START_RANGE..LottoConstants.END_RANGE }
```

Reply:
넵 공부해서 사용하도록 하겠습니다 !